app/views/admin/recommended_tags/index.html.haml

```diff
@@ -0,0 +1,31 @@
+%h1 Tag Manager
```

Comment:
Should this be Recommended Tag Manager?

Reply:
The designs have "Tag Manager" for both tabs so I left it it at that

https://zube.io/files/smartlogic/1c3a710262dbb20f1cd5a643474f9f37-image.png